team_admin/discord.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Discord Server
+---
+
+# Discord Server
+
+To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) communication both for communication with blue-shirts and among your team.
+
+If you do not wish to, or are unable to join Discord, you can still get support from <{{ site.emails.teams }}> as normal.
+
+## Joining
```

Comment:
Do you need to create a Discord account before trying to join our server? Perhaps mention that that's the case if so?

Reply:
The invite takes the form of a link, which will prompt you to create an account if you don't have one already.

Reply:
Cool. I think we should say that you'll be prompted to create an account if you don't already have one.